testsuite/common/src/main/resources/io/smallrye/metrics/base-metrics.properties

```diff
@@ -30,7 +30,7 @@ gc.%s.count.multi: true
 gc.%s.count.description:  Displays the total number of collections that have occurred. This attribute lists -1 if the collection count is undefined for this collector.
 gc.%s.count.mbean: java.lang:type=GarbageCollector,name=%s/CollectionCount
 gc.%s.time.displayName: Garbage Collection Time
-gc.%s.time.type: counter
+gc.%s.time.type: gauge
```

Comment:
I don't think this is correct - as it is the accumulated time, so only going up

Reply:
a counter has no unit while this metric expressed a time in milliseconds.